corehq/apps/app_manager/static/app_manager/js/releases/releases.js

```diff
@@ -293,7 +293,6 @@ hqDefine('app_manager/js/releases/releases', function () {
 
         self.goToPage = function(page){
             self.fetchState('pending');
-            self.savedApps([]);
```

Comment:
Is this fixing flickering or something?

Reply:
Yes, if we empty the list, it collapses completely and just `...loading` shows. If we don't empty the list, the current behaviour continues (i.e. `...loading` shows at the top, then the items are replaced)